New Crowdin translations (#169)
w3f/polkadot-wiki: docs/polkadot/node/governance/index.md

Polkadot uses a sophisticated governance mechanism that allows it to evolve gracefully over time at the ultimate behest of its assembled stakeholders.

To do this, we bring together various novel mechanisms, including an amorphous state-transition function stored on-chain and defined in a platform-neutral intermediate language (i.e. WebAssembly) and several on-chain voting mechanisms such as referenda with adaptive super-majority thresholds and batch approval voting.

All changes to the protocol must be agreed upon by stake-weighted referendum; the majority of the stake can always command the network.

## Mechanism

> In order to make any changes to the network, the idea is to compose of active token holders and council together to administrate the network upgrade decision. No matter whether the proposal is proposed by the public(DOTs holders) or council, it finally will have to go through the [referenda](#referenda) to let all DOTs holders make the decision.

Reviewer: agree

> In order to make any changes to the network, the idea is to compose of active token holders and council together to administrate the network upgrade decision. No matter whether the proposal is proposed by the public(DOT holders) or council, it finally will have to go through the [referenda](../../learn/governance.md#referenda) to let all DOT holders make the decision.

Reviewer 2: ```suggestion
In order to make any changes to the network, the idea is to compose active token holders and council together to administrate the network upgrade decision. No matter whether the proposal is proposed by the public (DOT holders) or council, it finally will have to go through the [referenda](../../learn/governance.md#referenda) to let all DOT holders make the decision.
```

The following steps are the governance procedure in the Polkadot network:

> - [Proposing proposals](#proposing-proposals) (Involved info: [Referenda](#referenda))

Reviewer: agree

- [Voting in a referendum](#voting-in-a-referendum) (Involved info: [Voluntary Locking](#voluntary-locking)

> - [Vote Counting](#vote-counting) (Involved info: [Adaptive Quorum Biasing](#adaptive-quorum-biasing))

Reviewer: yeah, it sounds better

> To better understand how council forms, please read [this section](#council).

Reviewer 2: ```suggestion
To better understand how the council is formed, please read [this section](#council).
```

> ### Proposing proposals

Reviewer 3: Looks like they are going with "referenda" according to this very recent commit from Joshy - 

https://github.com/paritytech/substrate/commit/7e7192c0f48e7ca97bc54327df55a72ccd2d479c

> ### Proposing proposals

Reviewer 2: good catch! @ansonla3 did you see this comment?

> ### Proposing proposals

Reviewer: @laboon oh. thanks! :)

> - **Public**: Anyone can propose a referenda by depositing minimum value in a certain period (No. of Blocks). If someone likes the proposal, they could deposit the same amount of tokens to support it. The one with the highest number of supported proposal will be selected to be a referendum. Those tokens will be released once the proposal is tabled. At genesis, every two weeks will have a referendum on the most supported proposal.

Reviewer 2: ```suggestion
- **Public**: Anyone can propose a referenda by depositing the minimum amount of DOTs for a certain period (No. of Blocks). If someone likes the proposal, they may deposit the same amount of tokens to support it. The proposal with the highest number of support bond will be selected to be a referendum. The bonded tokens will be released once the proposal is tabled. At genesis, every two weeks there will be a referendum for the most supported proposal.
```

- **[Council](#council)**: 

>     Unanimous Council - Where everyone in the council thinks that is a good proposal, it can move to the referenda.

Reviewer 2: ```suggestion
    Unanimous Council - When all members of the council agrees on a proposal, it can be moved to a referendum.
```

>     Majority Council - Where simple-majority council agreement must have more aye votes to nay votes to be accepted.

Reviewer 2: ```suggestion
    Majority Council - When agreement from only the simple majority of council members is needed. (More Aye votes to Nay votes for acceptance, more Nay votes to Aye votes for rejection.) 
```

Publicly submitted referenda, being `public`, can easily include malevolent or ill-considered actions. Here the onus must be placed on the proponents and so we bias any abstention votes against the motion, in favor of the (assumed safe, since its functional enough to administer this vote) status quo and use a `positive turnout bias`.

**Super-Majority Approve**

A ``positive turnout bias``, whereby a heavy super-majority of aye votes is required to carry at low turnouts, but as turnout increases towards 100%, it becomes a simple- majority-carriers as below.
$${against \over \sqrt{voters}} < {approve \over \sqrt{electorate}}$$

**Super-Majority Against**

A ``negative turnout bias``, whereby a heavy super-majority of nay votes is required to reject at low turnouts, but as turnout increases towards 100%, it becomes a simple-majority-carriers as below.
$${approve \over \sqrt{voters}} < {against \over \sqrt{electorate}}$$

**Simple-Majority**

Majority-carries, a simple comparison of votes, if there are more aye votes than nay, then the proposal is carried.
$${approve} > {against}$$

> To calculate the voting for a proposal, we need this four information.

Reviewer 2: ```suggestion
We need the following information to calculate the required voting thresholds of proposals:
```

> ```

Reviewer 2: Wouldn't this be better as a simple markdown list?

Approve - the number of aye votes

Against - the number of nay votes

Voters - the total number of voting tokens

Electorate - the total number of DOTs tokens issued in the network
```

We use the public proposal as an example so `Super-Majority Approve` formula will be applied. There is no strict quorum, but super-majority required increases as turnout lowers. For simplicity, assume we only have 1,000 DOTs tokens in total.
```
John  - 500 DOTs
Peter - 100 DOTs
Lilly - 150 DOTs
JJ    - 150 DOTs
Ken   - 100 DOTs

John: Votes `Yes`for a 2 week lock period  => 500 * 1 = 500 Votes

Peter: Votes `Yes` for a 2 week lock period => 100 * 1 = 100 Votes

JJ: Votes `No` for a 6 week lock period => 150 * 3 = 450 Votes
```

> $${450 \over \sqrt{1050}} < {600 \over \sqrt{1000}}$$

Reviewer 2: Where do the numbers 1050 and 1000 come from?

Reviewer: 1050 = the total number of votes for this proposal
1000 =  the number of tokens issued in this network

## Voluntary Locking

Polkadot utilize an idea called `Voluntary Locking` that lets the token holders increasing their voting power by declaring how long they are willing to lock-up their DOTs, hence, the maximum number of votes for each token holder will be calculated by the following formula:

```
Max votes = tokens * periods 
```

Based on the current testnet setting, the maximum number of lock periods could NOT be more than 6.

**Each period takes 2 weeks, which means the longest lock period would be 12 weeks.**

## Adaptive Quorum Biasing

Polkadot introduces a concept "Adaptive Quorum Biasing", which functions as a lever that the council can use to alter the effective super-majority required to make it easier or more difficult for a proposal to pass in the case that there is no clear majority of voting power backing it or against it.

![](../../../img/governance/adaptive-quorum-biasing.png)

Let's use the above image as an example.

> if there is publicly submitted referenda only has 25% turnout, the tally of "aye" votes has to reach 66% for it to pass since we applied the ``Positive Turnout Bias``.

Reviewer 2: ```suggestion
If there is publicly submitted referenda only has 25% turnout, the tally of "aye" votes has to reach 66% for it to pass since we applied the ``Positive Turnout Bias``.
```

### How to be a council member?

![](../../../img/governance/approval-vote.png)

At genesis, there will be 6 to 12 seats to start. For every two weeks, one of those seats is up for election and increase over the course of 9 months to 24 people (roughly one extra individual coming on every two weeks). All members have a fixed term (1 year). To elect a new council member, Polkadot employs `approval voting` method to allow token holders that choose a list of candidates they want to support in equal weight and the one with the most approval votes wins the election, while top-N runners-up remain on the candidates' list for next election.

Basically, instead of using one person one vote, [approval voting](https://en.wikipedia.org/wiki/Approval_voting) is a more expressive way to indicate their views. Token holders can treat it as boolean voting to support as many candidates as they want.

Let's take a look at the example below.
```
                                Round 1

>                                 Candidates

Reviewer 2: Wouldn't it be better to use a [markdown table](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#tables) for this?

                |   A   |   B   |   C   |   D   |   E   |

Token Holders

Peter               X               X       X       X

Alice                       X

Bob                                 X       X       X

Kelvin              X               X

                    2       1       3       2       2
```
The above example shows that candidate C wins the election in round 1, while candidate A, B, D & E keep remaining on the candidates' list for the next round.
```
                                Round 2

>                                 Candidates

Reviewer 2: Same as above regarding markdown table

### Tallying

Depending on who are the entity proposed the proposal and whether all council members voted yes, there are three different scenarios. 

We assume that a majority `council` agreement, with no veto, signals a sensible, perhaps an irregular state transition. For this, we use the `majority carries` metric. As an exception to this when there is **complete agreement** within the council we assume that is signals a largely technocratic and uncontroversial protocol change. For this reason we assert that the "burden of proof" should fall on those against the motion and thus we use the `negative turnout bias` metric.

Publicly submitted referenda, being `public`, can easily include malevolent or ill-considered actions. Here the onus must be placed on the proponents and so we bias any abstention votes against the motion, in favor of the (assumed safe, since its functional enough to administer this vote) status quo and use a `positive turnout bias`.

**Super-Majority Approve**

A ``positive turnout bias``, whereby a heavy super-majority of aye votes is required to carry at low turnouts, but as turnout increases towards 100%, it becomes a simple-majority-carriers as below.

> $${against \over \sqrt{voters}} < {approve \over \sqrt{electorate}}$$

Reviewer 2: I think we should better define what this equation means and where it comes from before dropping it here. One or two sentences that explain what each of these parameters mean and where they come from. Similar to the code comment [here](https://github.com/paritytech/substrate/blob/master/srml/democracy/src/vote_threshold.rs#L73).  Also would be good to link to where in the code the equation comes from and what module as the source of the equation.

Reviewer: umum... I moved the description to the above & rephasing a bit to see would be better or not.

$${approve \over \sqrt{voters}} < {against \over \sqrt{electorate}}$$

**Simple-Majority**

Majority-carries, a simple comparison of votes, if there are more aye votes than nay, then the proposal is carried.
$${approve} > {against}$$

We need the following information to calculate the required voting thresholds of proposals:

```
Approve - the number of aye votes

Against - the number of nay votes

Voters - the total number of voting tokens

Electorate - the total number of DOTs tokens issued in the network
```

We use the public proposal as an example so `Super-Majority Approve` formula will be applied. There is no strict quorum, but super-majority required increases as turnout lowers. For simplicity, assume we only have 1,500 DOTs tokens in total.
```
John  - 500 DOTs
Peter - 100 DOTs
Lilly - 150 DOTs
JJ    - 150 DOTs
Ken   - 600 DOTs

John: Votes `Yes`for a 2 week lock period  => 500 * 1 = 500 Votes

Peter: Votes `Yes` for a 2 week lock period => 100 * 1 = 100 Votes

JJ: Votes `No` for a 6 week lock period => 150 * 3 = 450 Votes
```

$${450 \over \sqrt{1050}} < {600 \over \sqrt{1500}}$$

> $${13.887} < {15.492}$$

Reviewer 2: Looks good :+1:

> Based on the above result, proposal will be approved. In addition, only the winning voter's tokens are locked, which means if that referenda hurts the network, then those who voted against it can immediately get their locked tokens back. They can exit the network and sell their tokens to the market before the proposal becomes effective. Moreover, winning proposals are autonomously enacted only after some cool-down period.

Reviewer 2: ```suggestion
Based on the above result, the proposal will be approved. In addition, only the winning voter's tokens are locked, which means if that referendum hurts the network, then those who voted against it can immediately get their locked tokens back. They can exit the network and sell their tokens to the market before the proposal becomes effective. Moreover, winning proposals are autonomously enacted only after some cool-down period.
```

## Voluntary Locking

> Polkadot utilize an idea called `Voluntary Locking` that lets the token holders increasing their voting power by declaring how long they are willing to lock-up their DOTs, hence, the maximum number of votes for each token holder will be calculated by the following formula:

Reviewer 2: ```suggestion
Polkadot utilizes an idea called `Voluntary Locking` that allows token holders to increase their voting power by declaring how long they are willing to lock-up their DOTs, hence, the maximum number of votes for each token holder will be calculated by the following formula:
```

```
Max votes = tokens * periods 
```

> Based on the current testnet setting, the maximum number of lock periods could NOT be more than 6.

Reviewer 2: ```suggestion
Based on the current testnet setting, the maximum number of lock periods is set to 6.
```

To elect a new council member, Polkadot employs `approval voting` method to allow token holders that choose a list of candidates they want to support in equal weight and the one with the most approval votes wins the election, while top-N runners-up remain on the candidates' list for next election. 

Basically, instead of using one person one vote, [approval voting](https://en.wikipedia.org/wiki/Approval_voting) is a more expressive way to indicate their views. Token holders can treat it as boolean voting to support as many candidates as they want.

Let's take a look at the example below.

||||Round 1|||
||:----:|:---:|:---:|:---:|:---:|
|**Token Holders**|||**Candidates**|||
||A|B|C|D|E|
|Peter|X||X|X|X|X|
|Alice||X|||||
|Bob|||X|X|X|
|Kelvin|X||X|||
|**Total**|2|1|3|2|2|

The above example shows that candidate C wins the election in round 1, while candidate A, B, D & E keep remaining on the candidates' list for the next round.

|||Round 2|||
||:----:|:---:|:---:|:---:|
|**Token Holders**||**Candidates**||
||A|B|D|E|
|Peter|X|X|||
|Alice|X|X|||
|Bob|X|X|X|X|
|Kelvin|X|X|||
|**Total**|4|4|1|1|

> For the top-N(say 4 in this example) runners-up, they can stay there and keep their votes persist until next election. After round 2, even though candidate A & B get the same votes in this round, candidate A gets elected because after adding the older unused approvals counter still higher than B.

Reviewer 2: ```suggestion
For the top-N (say 4 in this example) runners-up, they can remain and their votes persist until the next election. After round 2, even though candidates A & B get the same number of votes in this round, candidate A gets elected because after adding the older unused approvals, it is higher than B.
```

This would be the tentative governance configuration for Polkadot in the initial genesis. It will be changed if any security loopholes have been found after third-party auditing.

### Resources

- [Governance Description](../../learn/governance.md#referenda)
- [Democracy Module](https://github.com/paritytech/substrate/tree/master/srml/democracy/src)

## [Usage of DOT](../../learn/DOT.md#dots-for-governance)

## Guides

### [How to create a proposal]()
### [How to join the council]()

> ### [How to propose a referenda]()

Reviewer 2: Will this be different than `How to create a proposal`?

Reviewer: sounds remove ``How to create a proposal`` and use ``How to propose a referendum`` would be better

According to the above scenario, even though combining both Logan and Kevin's DOTs is more than Peter, the lock period for both of them is far less than Peter, leading to their voting power counting as less.

### Tallying

> Depending on who are the entity proposed the proposal and whether all council members voted yes, there are three different scenarios. We can use following table for reference.

Reviewer 2: ```suggestion
Depending on which entity proposed the proposal and whether all council members voted yes, there are three different scenarios. We can use following table for reference.
```
